src/index.js

```diff
@@ -23,6 +25,12 @@ export default class AddressAutocomplete {
       );
     }
 
+    if (options) {
```

Comment:
Instead of checking if ~~`options`~~ `optionsOrCallback` exists, let's check to see if `optionsOrCallback` is a `function`. If `typeof optionsOrCallback === "function"`, then we know we do not have any additional options and that is the callback. However, if `typeof optionsOrCallback === "object"`, then we know additional options are being passed and we need to merge those. Also, we know the third parameter is the callback in this scenario since options are passed in as the third parameter. If the second param is not an object or a function, then we need to `throw` and error and let the developer know that they have entered an incorrect type and we only allow `object` or `function`.

Also, instead of merging in the options on line 64, we can go ahead and merge those in now and set `this.options`. Then we can just pass that in on line 66: 
```js
this.autocomplete = new google.maps.places.Autocomplete(this.element, this.options);
```